src/chrome/content/rules/NIIF.xml

```diff
@@ -1,14 +1,13 @@
-
 <!--
-Disabled by https-everywhere-checker because:
-Fetch error: http://www.hboneplus.hu/ => https://www.hboneplus.hu/: (60, 'SSL certificate problem: certificate has expired')
-
+	Expired:
+		- www.hboneplus.hu
 -->
-<ruleset name="NIIF Int&#233;zet (partial)" default_off='failed ruleset test'>
-	<target host="www.hboneplus.hu" />
+
+<ruleset name="NIIF Int&#233;zet (partial)">
+	<target host="niif.hu" />
 	<target host="www.niif.hu" />
 	<target host="www.ca.niif.hu" />
+	<target host="keys.niif.hu" />
```

Comment:
Would you like to search for additional subdomains?

Reply:
@J0WI No, the point of this PR is to add the keyserver subdomain to this ruleset.